webpack/assets/javascripts/foreman_tools.js

```diff
@@ -87,6 +87,9 @@ export function updateTable(element) {
   let uri = new URI(window.location.href);
   let search, perPage;
 
+  if (getQueryStringValue(uri, 'page') > 1) {
+    uri.setSearch('page', 1);
+  }
```

Comment:
won't this always set the page to 1 regardless? afaiu this function is called both from the per_page and the search submit?.

can you write a test to demonstrate the various options?